### PR TITLE
Core: correct Other as 8000 and Books as 7000

### PR DESCRIPTION
**Pull request by garfield69**
and update any indexers affected
as documented in https://newznab.readthedocs.io/en/latest/misc/api/#predefined-categories

### CODE REVIEW

**Review by ngosang — APPROVED** [review]
LGTM. Did you test Readerr?

**Comment by garfield69** [comment]
yes, I've tested it with Readerr. 
I should note however that existing Readerr users that are using a Jackett indexer will need to update their Readerr indexer's config to specify the correct new book categories. 
For instance, with ebookparadijs, Readerr was configured to use 8010, but once changed to 7010 it worked fine.

**Comment by garfield69** [comment]
Also tested this with Sonarr, which recognises that magnet4you only has category Other and allows that Sonarr indexer to be saved and used. tested OK.

**Comment by ilike2burnthing** [comment]
May be worth pinning an issue notifying users once this is merged (even though some people never read them).

**Comment by garfield69** [comment]
Hah. there is a maximum of 3 tickets that can be pinned at any one time, as we are already at that limit ;-)

File: src/Jackett.Common/Definitions/brasiltracker.yml [hunk 1]
```diff
@@ -26,14 +24,10 @@ settings:
...
-    type: select
-    label: The BrasilTracker web site does not provide categories. Select the category you want Jackett to set on all results returned.
-    default: 3
-    options:
-      1: TV
-      2: Movies
-      3: Other
+  - name: info_8000
+    type: info
+    label: About BrasilTracker Categores
```
Comment (Taloth) [inline src/Jackett.Common/Definitions/brasiltracker.yml:29]: Categores -> Categories? (And the others too)

**Pull request merged**